chapters/qa-6.mdx

```diff
@@ -37,17 +37,17 @@ Občas nastane chyba - pokusíš se změnit oběd na příští týden příliš
 
 ### Stavové kódy
 
-Při testování aplikací se může stát, že požadavek (např. na logo stránky) selže a je vhodné vědět, proč se tomu tak stalo. Pro snadné určení výsledku požadavku je součástí HTTP (a také HTTPS) stavový kód. Stavových kódů je mnoho, ale podstatných je pro práci jen pár.
+Při testování aplikací se může stát, že požadavek (např. na logo stránky) selže a je vhodné vědět, proč se tomu tak stalo. Pro snadné určení výsledku požadavku je součástí HTTP (a také HTTPS) stavový kód. Stavových kódů je mnoho, ale podstatných je pro práci jen pár.
 
-Ty důležité jsou popsány na [Strafelda.cz - Co je stavový kód](https://www.strafelda.cz/stavovy-kod), pokud by ses setkal s nějakým zde neuvedeným můžeš využít i [Wikipedii - Stavové kódy](https://www.wikiwand.com/cs/Stavov%C3%A9_k%C3%B3dy_HTTP).
+Ty důležité jsou popsány na [Strafelda.cz - Co je stavový kód](https://www.strafelda.cz/stavovy-kod), pokud by ses setkal s nějakým zde neuvedeným můžeš využít i [Wikipedii - Stavové kódy](https://www.wikiwand.com/cs/Stavov%C3%A9_k%C3%B3dy_HTTP).
 
 ## DevTools - záložka Network
 
-V DevTools můžeš zjistit mnoho informací o požadavku například jeho výsledek a stavový kód. Prozkoumáme cvičný web [https://astronauts.vercel.app/](https://astronauts.vercel.app/). Abys dosáhl stejného výsledku jako my na obrázcích je potřeba drobná kosmetická úprava pomocí DevTools. Změň u tagu _img_ atribut _src_ z _rocket.png_ na _rocket2.png_. Poté přejdi do záložky **Network**.
+V DevTools můžeš zjistit mnoho informací o požadavku například jeho výsledek a stavový kód. Prozkoumáme cvičný web [https://astronauts.vercel.app/](https://astronauts.vercel.app/). Abys dosáhl stejného výsledku jako my na obrázcích je potřeba drobná kosmetická úprava pomocí DevTools. Změň u tagu _img_ atribut _src_ z *rocket.png* na _rocket2.png_. Poté přejdi do záložky **Network**.
```

Comment:
why does the rocket.png have a different emphasis than the rocket2.png?

Reply:
It is enforced automatically, but nice catch!